src/main/java/physic/PhysicManager.java

```diff
@@ -0,0 +1,87 @@
+package physic;
+
+import geometry.Point;
+import geometry.Rectangle;
+import object.GameObject;
+import object.GameObjectManager;
+
+import java.util.Collection;
+import java.util.HashSet;
+
+public class PhysicManager {
+
+    private Rectangle field;
+
+    private GameObjectManager gameObjectManager;
+
+    public PhysicManager(Rectangle field, GameObjectManager gameObjectManager) {
```

Comment:
PhysicManager does not need to use gameObjectManager. It should use only game objects collection.

Reply:
should I remove GameObjectManager from project than? It's mostly acting as a Collection

Reply:
I think so. I've noticed this before.